### ISSUE TITLE
The Lets Encrypt ACME server was probably unable to reach http://<sitename>.com/.well..

### ISSUE THREAD

**Issue by romitgirdhar** [sjkp/letsencrypt-siteextension#123]
Hello,

I know this is probably redundant, but, I've read through all the issues here and am still facing this issue. I'm unable to get this working. I keep getting the following error:

The Lets Encrypt ACME server was probably unable to reach http://<site-name>.com/.well-known/acme-challenge/_yW88K8i-TJnXTW4BgG01psKmj7JmJeVSZhCOl_V7n8 view error report from Lets Encrypt at https://acme-staging.api.letsencrypt.org/acme/authz/5z1_aFYbzp50nPgdeNYfXhYajtN2XzNJVNSfEReZBbI for more information

When I click on the link the extension is trying to reach, it takes me to a page that has the text "Not Found". 

I have a nodeJS based app. I've created the Service Principal and ensured it has access to my web app. I've also add this additional code snippet to my app.js: `app.use('/.well-known', express.static('.well-known'));`

I also went into the Azure web app console (via Kudu) to check if the extension was creating the '.well-known' folder, but, it hasn't. Just to ensure that my app is able to serve static content, I even tried pulling one of the images from my website by going to: https://<site-name>.com/images/image1.jpg and the request was successful.

I would appreciate any help here :)

Thank you!

**Comment by romitgirdhar** [comment]
I just noticed that the '.well-known' folder gets created in my 'wwwroot' folder in the Azure web app. Looks like the problem was with this line of code: `app.use('/.well-known', express.static('.well-known'));`. Just needed to add __dirname to ensure it was loading an absolute path.

Here's the code I had to add: `app.use('/.well-known', express.static(path.join(__dirname, '.well-known')));`

Thank you!
